lib/config.js

```diff
@@ -56,76 +56,73 @@ config.paths = {
   log: null
 };
 
-// logging
+/* logging options
+ * formatter options:
+ *  default, json, logstash or a custom formatter function
+ */
 config.loggers = {};
 
 // transport for console logging
 config.loggers.console = {};
+
 config.loggers.console.level = 'debug';
 config.loggers.console.silent = false;
-config.loggers.console.json = false;
-config.loggers.console.timestamp = true;
-config.loggers.console.colorize = true;
 // bedrock options
 config.loggers.console.bedrock = {};
-// move 'module' meta to a pretty message prefix if available
-config.loggers.console.bedrock.modulePrefix = true;
+config.loggers.console.bedrock.formatter = 'default';
+config.loggers.console.bedrock.colorize = true;
```

Comment:
Are these going to be breaking changes? I was hoping to avoid this being a breaking release.

Reply:
@dlongley I was building this out with the intention of it being a breaking change.  The most important reason why this is the case is the fact that the Bedrock API is designed such that configuration options are being sent directly into Winston.  For instance `config.loggers.console` is passed directly into the `console` logger.  There have been many breaking changes in the Winston configuration API.  For instance, `console.json = true` and `console.colorize = true` is no longer a thing.  All those options are configured by way of formatters now.  Writing a translation from 1.x features to 2.x/3.x is not an option as far as I'm concerned.

Furthermore, I think some features got built into some of the custom Transports that have never seen any use.  For instance there was some `preformatting` directives that have never been used to my knowledge.  I would like to dump those legacy behaviors because it only serves to slow things down if it's not useful functionality.